tools/sandbox/application.py

```diff
@@ -157,8 +157,8 @@ def analyze(self) -> None:
                 line = line.rstrip()
                 if line == "":
                     break
-                elif "DEBUG" in line:
-                    continue
+                elif "INFO" in line or "ERROR" in line:
+                    emit("pysa_results_channel", {"type": "output", "line": line})
                 elif "ERROR" in line and "is not part of the environment" in line:
```

Comment:
The body of this `elif` is now unreachable. The above `elif` will be entered if `line` contains `ERROR`, so we'll never make it into this one that is also checking for `line` to contain `ERROR`.

Reply:
My bad, reordered now.